tools/WebhookProcessor/github_webhook_processor.php

```diff
@@ -359,12 +359,31 @@ function check_dismiss_changelog_review($payload){
 				dismiss_review($payload, $R['id'], 'Changelog added/fixed.');
 }
 
+function check_webeditor($payload, $opening) {
+	if(is_maintainer($payload, $payload['pull_request']['user']['login']))
+		return;
+	$githubgpgkeyid = '04AEE18F83AFDEB23';
+	$commits = json_decode(apisend($payload['pull_request']['commits_url']), TRUE);
+	foreach($commits as $commit) {
+		if(!$commit['commit']['verification']['verified'])
+			return false;
+		$sig = $commit['commit']['verification']['signature'];
+		$keyid = substr(bin2hex(base64_decode(substr(str_replace('\n', '', $sig),53,12))),0,-1);
+		if(strtoupper($keyid) != $githubgpgkeyid)
+			return false;
+	}
+	if($opening)
+		create_comment($payload, 'Pull requests made via the GitHub web editor are guarranteed to not be tested and only prove to shame you as coder.');
```

Comment:
i used to copypaste changes from local after testing them into the webeditor to PR them since i was too lazy to go through git

Reply:
good practice
thanks for repo bloat

Reply:
that is the most insane thing you've ever said

Reply:
yeah it was BAD and i dont do it anymore